src/test/java/acceptance/queue/QueueSteps.java

```diff
@@ -229,6 +264,7 @@ public void the_client_should_display_to_console(List<OutputRepresentation> expe
             assertThat(output, containsString(expectedLine.output));
         }
         System.out.println(output);
+        queueBasedImplementationRunner.stop();
```

Comment:
Why do we need this here?

Reply:
Explanation in https://github.com/julianghionoiu/tdl-client-java/pull/7#discussion_r222256321